src/geheimtur/impl/oauth2.clj

```diff
@@ -1,6 +1,7 @@
 (ns geheimtur.impl.oauth2
   (:require [clj-http.client :as client]
             [clojure.walk :refer [keywordize-keys]]
+            [clojure.string :as str]
```

Comment:
This import is not needed and it redefines [clojure.core/str](https://clojuredocs.org/clojure.core/str).